resources/documentation/multiple_icebergs.rst

```diff
@@ -227,45 +291,226 @@ And add management2 to iceberg2, by editing /etc/bluebanquise/inventory/cluster/
+its iceberg, and be able to deploy the configuration on its iceberg nodes.
+
+We need to ensure mngt2 is part of a group that will mount the repositories and
+bluebanquise, in nfs.yml. By default, this group is called
+*secondary_managements*.
+
+Create file *inventory/cluster/groups/secondary_managements* with the following
+content:
+
+.. code-block:: text
+
+  [secondary_managements]
+  mngt2
+
+Then ensure in file *inventory/group_vars/all/general_settings/nfs.yml* you have
+exactly at least these two exports:
+
+.. code-block:: yaml
+
+  nfs:
+
+    ...
+
+    bluebanquise:
+      mount: /etc/bluebanquise
+      export: /etc/bluebanquise
+      server: mngt1
+      clients_groups:
+        - secondary_managements
+      take_over_network: ice1-1
+      export_arguments: ro,no_root_squash,sync
+      mount_arguments: ro,intr,nfsvers=4.2,bg
+
+    repositories:
+      mount: /var/www/html/repositories
+      export: /var/www/html/repositories
+      server: mngt1
+      clients_groups:
+        - secondary_managements
+      take_over_network: ice1-1
+      export_arguments: ro,no_root_squash,sync
+      mount_arguments: ro,intr,rsize=32768,wsize=32768,nfsvers=4.2,bg
+
+So mngt1 will export both folders, and members of secondary_managements (so
+mngt2) will mount it.
+
+.. note::
+  All is set to read only here (ro). It is up to you to switch to read write if
+  needed.
 
-Play again playbooks on management1, so management2 is added into dhcp, pxe, dns, hosts, etc. configuration files.
+If you just added these new nfs exports, play the role nfs_server on mngt1 and
+check that mngt1 now export these foldes, using *showmount -e mngt1* command.
 
-Once done, use standard pxe boot procedure to deploy OS on management2.
+Next, we will need a playbook for mngt2. Copy current mngt1 dedicated playbook:
 
-Now, few steps has to be followed in a specific order in order to deploy configuration on it.
+.. code-block:: text
+
+  cp /etc/bluebanquise/playbooks/mngt1.yml /etc/bluebanquise/playbooks/mngt2.yml
 
-Deploy management configuration
--------------------------------
+And change target host inside to match mngt2.
 
-First, it is needed that management2 mount over nfs the repositories and the BlueBanquise configuration from management1.
+From now, few steps need to be done in a very strict order. We are going to
+force mngt2 to be part of iceberg1 for few commands, in order to be able to
+bootstrap it. To do so, we will execute the mngt2.yml playbook, with an extra
+variable, that will force *j2_current_iceberg* to be **iceberg1**.
 
-Ensure management2 is part of a group that will mount the repositories and ansible, in nfs.yml. If no, you can create a custom group for that, that would regroup all non top iceberg management.
+.. image:: images/multiple_icebergs_3.svg
 
-Copy current management1 playbook:
+We need first mngt2 to be able to install packages, and so to use mngt1 as
```

Comment:
Shouldn't mngt2 always use mngt1 in its dnf configuration? Unless one want to strictly isolate this iceberg, but the change should be part of the isolation process.

Reply:
Interesting comment. Yes, I missed that point. DNS chain need some review I agree.